src/ui/Controllers/RequestAccessController.cs

```diff
@@ -1,48 +1,43 @@
-using System.Linq;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
 using GovUk.Education.ManageCourses.ApiClient;
 using GovUk.Education.ManageCourses.Ui.ViewModels;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
-namespace GovUk.Education.ManageCourses.Ui.Controllers
-{
-    public class RequestAccessController : CommonAttributesControllerBase
-    {   
+namespace GovUk.Education.ManageCourses.Ui.Controllers {
```

Comment:
This file should be reformatted to the usual c# layout.

Reply:
Fixed